.github/workflows/ci-build.yml

```diff
@@ -20,34 +15,34 @@ jobs:
       - uses: actions/setup-java@v2
         with:
           distribution: 'zulu'
```

Comment:
In my last review (different project) I think you used a different one? Is there some criteria?

Reply:
Good find. I copied the snipped from the wrong place.